luatest/server.lua

```diff
@@ -297,6 +412,11 @@ end
 function Server:start(opts)
     checks('table', {wait_until_ready = '?boolean'})
 
+    opts = opts or {}
+    if self.config_file and opts.wait_until_ready == nil then
+        opts.wait_until_ready = self.net_box_uri ~= nil
+    end
```

Comment:
Modifying a user-provided options table is not a good citizen behavior. See https://github.com/tarantool/crud/issues/192 for example of a real world problem caused by it.

Reply:
Fixed.